05_iCommands_Handson_User-Training.md

```diff
@@ -216,229 +221,217 @@ Let’s try to understand what this means. The example.txt that we uploaded to i
 
-#### File Bundling
-To upload and download structured files such as tar files to/from iRODS, we can use `ibun` command.
+As good practice we can tag the tar file using the `-Dtar` flag when uploading the file with `iput`. Alternatively, this 'dataType' tag can be added later with the `isysmeta` command: `isysmeta mod /path/to/tarfile.tar datatype 'tar file'`.
 
-A tar file containing many small files can be created with normal unix tar command on our local machine . We can then upload the tar file to the iRODS server as a normal iRODS file. Furthermore, `ibun -x` command can be used to extract/untar the uploaded tar file. The extracted subfiles and subdirectories will then be appeared as normal iRODS data objects and sub-collections.
+To illustrate, let's first create a small folder with files and tar it.
 
-As a good practice we can tag the tar file using the -Dtar flag when uploading the file with `iput`. The dataType tag can be added later with the isysmeta command. For example:
-isysmeta mod /kuleuven_tier1_pilot/home/vsc33586/mydir.tar datatype 'tar file'
+```
+mkdir fortar
+cd fortar
+for file in one two three four; do touch $file.txt; done
+cd ../
+tar -chlf test.tar fortar
+```
+
+Running `ls fortar` will show its contents. Now we can send the file to iRODS and untar it with `ibun -x` into a collection called 'test_collection'.
 
 ```sh
-  tar -chlf test.tar –C myCollection my file
-  iput -Dtar test.tar
-  ibun -x test.tar test_collection
+iput -Dtar test.tar
+ibun -x test.tar test_collection
 ```
 
-We can also add/bundle an iRODS collection into a tar file by using `ibun -c` command.
+We can also add/bundle an iRODS collection into a tar file with the `ibun -c` command.
 
 ```sh
-  ibun -cDtar test.tar myCollection1
+ibun -cDtar test2.tar test_collection
 ```
 
 ### Metadata
-Metadata is often called data about data and is used to facilitate data discovery to improve search and retrieval. iRODS provides the user with the possibility to create attribute-value-unit triples and store them with the data. The triples are stored in the iCAT catalogue.
 
-Metadata attribute-value-units triples (AVUs) consist of an Attribute-Name, Attribute-Value, and an optional Attribute-Units.  They can be added via the 'add' command (and in other ways), and then queried to find matching objects.
+Metadata, often called "data about data", is used to facilitate data discovery, search and retrieval. iRODS provides the user with the possibility to create attribute-value-unit triples attached to some data. The triples are stored in the iCAT catalogue.
+
+Metadata attribute-value-units triples (AVUs) consist of an Attribute-Name, Attribute-Value, and optionally Attribute-Units.
+They can be manipulated via the `imeta` command, followed by:
 
-For each command, -d, -C, -R, or -u is used to specify which type of object to work with: data objects (iRODS files), collections, resources, or users.
+- `add` to add a new AVU;
```

Comment:
Maybe best to add 'set' here (as you suggested yourself later)

Reply:
Ok! And its function would be to modify the value of all AVUs of an object that have the same attribute name, right? (In contrast to `mod`, which requires you to specify the full AVU and allows you to change any part of it)

Reply:
Yes, there are two things that -as far as I understand- imeta set does differently:

- It just has a simpler syntax
- It modifies all values with the same attributes (not sure how this interacts with units though)